package.json

```diff
@@ -33,7 +33,7 @@
     "date-fns": "^1.30.1",
     "material-icons": "^0.2.3",
     "moment": "^2.24.0",
-    "netlify-identity-widget": "github:iteatime/netlify-identity-widget",
+    "netlify-identity-widget": "^1.9.1",
```

Comment:
I think I've made some changes in translations in our repo since my PR on the official one. It needs some checks 😉

Reply:
https://github.com/netlify/netlify-identity-widget/pull/353